bare-metal/apple-silicon/troubleshooting/cant-connect-using-vnc.mdx

```diff
@@ -63,3 +108,7 @@ Make sure that either "Screen sharing" or "Remote management" are enabled on you
 ### Verify that incoming VNC connections are not blocked by the firewall
 
 Make sure that your Mac Mini M1 firewall allows incoming connections for your VNC client. Go to **System Settings** and then, depending on the OS version, **Firewall** will be located in either **Network** or **Security & Privacy**.
+
+### Further information
+
+* Official Apple documentation: [Block connections to your Mac with a firewall](https://support.apple.com/en-gb/guide/mac-help/mh34041/14.0/mac/14.0)
```

Comment:
I think this is unrelated to this doc because it explains how to block all traffic to a service, with no possibility to filter good/bad sources.